code/game/area/areas.dm

```diff
@@ -20,6 +20,8 @@
 	var/blob_allowed = TRUE // Does it count for blobs score? By default, all areas count.
 	var/clockwork_warp_allowed = TRUE // Can servants warp into this area from Reebe?
 	var/clockwork_warp_fail = "The structure there is too dense for warping to pierce. (This is normal in high-security areas.)"
+	var/abductor_warp_allowed = TRUE // Similar to above, for abductors! Mostly to protect the AI.
+	var/abductor_warp_fail = "This area is coated in bluespace interference, it's not possible to warp here safely!"
```

Comment:
Isn't there no tp flag or something used in wizard's den?

Reply:
That might remove all teleportation entirely instead of just abductor

Reply:
It does exist, is used in most of centcom

 Issue with reusing that variable is that it will prevent normal crew from tping to the AI sat, e.g. by the teleporter, since the beacon is in a protected area.

Reply:
Then just rename clockwork warp and reuse it

Reply:
That'd result in EVA, armory, tcomms, gravgen and..... chapel being blocked. Former make sense, latter would be weird.

Could be done if needed though

Reply:
okay I get you now, but the fail message could be unified instead of creating 2 similar vars

Reply:
just make it a list of banned areas